recourse_methods/mrmc_method.py

```diff
@@ -57,26 +57,26 @@ def __init__(self,
         
         Args:
             dataset: The dataset to provide recourse over.
-            preprocessor: The dataset's preprocessor.
+            adapter: The dataset's adapter.
```

Comment:
Nit: adapter: A RecourseAdapter object to transform the data between human-readable and embedded space.